src/sinks/aws_cloudwatch_logs/mod.rs

```diff
@@ -86,6 +89,15 @@ inventory::submit! {
     SinkDescription::new::<CloudwatchLogsSinkConfig>("aws_cloudwatch_logs")
 }
 
+#[derive(Clone, Copy, Debug, Default)]
+pub struct CloudwatchLogsDefaultBatchSettings;
+
+impl SinkBatchSettings for CloudwatchLogsDefaultBatchSettings {
+    const MAX_EVENTS: Option<usize> = Some(10_000);
+    const MAX_BYTES: Option<usize> = Some(1_048_576);
+    const TIMEOUT_SECS: NonZeroU64 = unsafe { NonZeroU64::new_unchecked(1) };
```

Comment:
We can _almost_ get rid of the `unsafe` here. Once Rust `1.57` lands, we can use a helper function like
```rust
const fn non_zero(val: u64) -> NonZeroU64 {
    match NonZeroU64::new(val) {
        Some(x) => x,
        None => panic!("Value must be non-zero!")
    }
}
```

I was hoping to find something that works today, but that's all I got

Reply:
Hmm, I'll add a TODO near the `BatchConfig` declaration so we don't forget this.